datahub-web-react/src/alchemy-components/components/Select/Nested/NestedSelect.tsx

```diff
@@ -147,6 +149,7 @@ export const selectDefaults: SelectProps = {
     isMultiSelect: false,
     width: 255,
     height: 425,
+    shouldManuallyUpdate: false,
```

Comment:
minor nit, but I don't think i would know what this means without digging much more. maybe we can rename to `shouldDisplayConfirmationFooter`?